README.md

```diff
@@ -38,6 +38,14 @@ in the P4_tutorial.pdf in the tutorial directory.
 
 A P4 Cheat Sheet is also available [online](https://drive.google.com/file/d/1Z8woKyElFAOP6bMd8tRa_Q4SA1cd_Uva/view?usp=sharing)
 which contains various examples that you can refer to.
+
+## P4 Documentation
+
+The documentation for P4_16, P4_14, P4 Runtime and Portable Switch Architecture(PSA) is available [here](https://p4.org/specs/)
+
+All excercises in this repository use the v1model, the documentation for which is available at:
```

Comment:
I think it would be good to say "the v1model architecture" instead of only "the v1model", in this sentence and at the end of numbered item 1. below.